### PR TITLE
Allow for providing defualt eslintConfig, closes #50

### LINKED ISSUE #50: Make eslintConfig the same way as prettierOptions
i.e. it <strike>should</strike> could provide just the defaults if filePath is provided.

It's in the same spirit as "You can also provide some of the options and have the remaining options derived via your eslint config.", but for eslint config, not prettier config.

### PR DESCRIPTION
**Pull request by sheerun**
I'm short on time. I think most of it is done, but could you fix any issues you find by yourself?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,7 +62,8 @@
       "max-len": [
         2,
         80
-      ]
+      ],
+      "complexity": ["error", 15]
```
Comment (kentcdodds) [inline package.json]: Definitely don't want to do this. There's a reason it's set to where it is.

File: src/utils.test.js [hunk 2]
```diff
@@ -1,4 +1,134 @@
...
+        quotes: [2, 'single', {avoidEscape: true, allowTemplateLiterals: true}],
+      },
+      options: {
+        parser: 'babylon',
+        printWidth: 120,
+        tabWidth: 2,
+      },
+    },
+  },
+  // Thanks, 100% coverage... ridiculous
```
Comment (kentcdodds) [inline src/utils.test.js]: Haha, well, at least we're more confident that it wont break ;)

File: src/utils.js [hunk 3]
```diff
@@ -5,7 +5,44 @@ import getLogger from 'loglevel-colored-level-prefix'
 const logger = getLogger({prefix: 'prettier-eslint'})
 
 /* eslint import/prefer-default-export:0 */
-export {getOptionsForFormatting}
+export {getOptionsForFormatting, defaultEslintConfig}
+
+function defaultEslintConfig(eslintConfig = {}, defaultConfig = {}) {
```
Comment (kentcdodds) [inline src/utils.js]: Why don't we simply this considerably by using [`lodash.merge`](https://www.npmjs.com/package/lodash.merge)? Could probably reduce the amount of tests required by a lot :)

**Comment by sheerun** [comment]
@kentcdodds Please refresh

**Comment by sheerun** [comment]
btw. could you release it soon?

**Comment by sheerun** [comment]
@kentcdodds Thanks to your fast merges I've managed to publish https://github.com/sheerun/prettier-standard :)

Thank you!

**Comment by kentcdodds** [comment]
Cool! Great work!

**Pull request merged**